app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt

```diff
@@ -68,11 +66,8 @@ class ProfileListAdapter(
             binding.profileCard.setOnClickListener {
                 val args = Bundle()
                 args.putParcelable("Profile", item)
-                val navOptions = NavOptions.Builder().setEnterAnim(R.anim.nav_default_enter_anim).setExitAnim(
-                    R.anim.nav_default_exit_anim
-                ).setPopEnterAnim(R.anim.nav_default_pop_enter_anim).setPopExitAnim(R.anim.nav_default_pop_exit_anim)
-                    .build()
-                Navigation.findNavController(it).navigate(R.id.detailsFragment, args, navOptions)
+                val dialog = newInstance(args)
```

Comment:
One small change.
change newInstance() to DetailsFragment.newInstance().

Reply:
@rob729  Ok, I will do it. Is everything else alright?